test/pytorch/test_models_recognition.py

```diff
@@ -49,7 +49,7 @@ def test_master(mock_vocab, max_len=50, batch_size=4):
     master = recognition.MASTER(vocab=mock_vocab, input_shape=(3, 32, 128))
     input_tensor = torch.rand((batch_size, 3, 32, 128))
     target = ["i", "am", "a", "jedi"]
-    logits = master(input_tensor, target)["out_map"]
+    logits = master(input_tensor, target, return_model_output=True, return_preds=True)['out_map']
```

Comment:
Considering what's implemented yet, you don't need the `return_preds=True` (it doesn't do anything)